### ISSUE TITLE
Sudden animation at 'News & Updates'

### ISSUE THREAD

**Issue by Schoeneh** [Public-Environmental-Data-Partners/website#164]
**Describe the bug**
Not really a bug, more a potential accessibility issue. While scrolling through the list of blogposts, a gif suddenly started animating: https://pedp-website.vercel.app/news-and-updates/2-212-studies-used-disappearing-federal-climate-justice-tools-we-analyzed-them-all

See here for more info re accessibility: https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html

**To Reproduce**
1. Go to 'News & Updates'
2. Scroll down to "Load More" & click
3. Scroll down until the blogpost "2,212 studies used disappearing federal climate justice tools. We analyzed them all." appears

**Expected behavior**
All other images are still, this one should also be still. Or if it needs to be animated, it should have a clear & easy way to start/stop the animation

**Screenshots**

<img width="1121" height="774" alt="Image" src="https://github.com/user-attachments/assets/c7e85f7f-faf8-4285-a64c-f76c71e02f96" />

**Desktop (please complete the following information):**
 - OS: Fedora 44
 - Browser: Firefox
 - Version: 154.0 (64-bit)

**Comment by Schoeneh** [comment]
Also, the GIF is pretty large compared to the other images; same issue as #163 applies here
